storage-account.tf

```diff
@@ -54,3 +54,11 @@ resource "azurerm_storage_container" "sscs_container" {
   resource_group_name  = "${azurerm_storage_account.storage_account.resource_group_name}"
   storage_account_name = "${azurerm_storage_account.storage_account.name}"
 }
+
+output "storage_account_name" {
+  value = "${azurerm_storage_account.storage_account.name}"
+}
+
+output "storage_account_primary_key" {
+  value = "${azurerm_storage_account.storage_account.primary_access_key}"
```

Comment:
same as above

Reply:
I've marked this one as sensitive because it is, but it doesn't do anything at all...

Reply:
it masks the value in the Jenkins log :)

Reply:
the pipeline will still show them